### ISSUE TITLE
Not compartible with new Akka 2.6.0

### ISSUE THREAD

**Issue by shestero** [enragedginger/akka-quartz-scheduler#89]
There is no akka.actor.ExtensionKey more...

**Comment by guntiso** [comment]
```
Symbol 'type akka.actor.ExtensionKey' is missing from the classpath.
This symbol is required by 'object com.typesafe.akka.extension.quartz.QuartzSchedulerExtension'.
```

**Comment by enragedginger** [comment]
This is addressed by #90. New build coming soon.

**Comment by enragedginger** [comment]
The new build is out and should be available on maven central shortly.
